openedx/core/lib/api/utils.py

```diff
@@ -0,0 +1,57 @@
+import functools
+from importlib import import_module
+
+from django.conf import settings
+
+
+def pluggable_override(override):
```

Comment:
One more comment:
It would be great if this general-purpose plugin functionality was co-located with our other general-purpose Plugin framework: https://github.com/edx/edx-django-utils/tree/master/edx_django_utils/plugins

Thanks!

Reply:
Thanks for pointing to this, @nasthagiri! Should it be in the `utils.py` module there or in a separate one?

> To make this proposal even stronger, I suggest having a recommendation for namespacing these new Global constants.

Good point, I'll add this :+1: